src/WorldStorage/NamespaceSerializer.cpp

```diff
@@ -131,6 +131,91 @@ namespace NamespaceSerializer
+			case mtEndermite:      return "minecraft:endermite";
+			case mtEvoker:         return "minecraft:evoker";
+			case mtFox:            return "minecraft:fox";
+			case mtGhast:          return "minecraft:ghast";
+			case mtGiant:          return "minecraft:giant";
+			case mtGuardian:       return "minecraft:guardian";
+			case mtHorse:          return "minecraft:horse";
+			case mtHoglin:         return "minecraft:hoglin";
+			case mtHusk:           return "minecraft:husk";
+			case mtIllusioner:     return "minecraft:illusioner";
+			case mtIronGolem:      return "minecraft:iron_golem";
+			case mtLlama:          return "minecraft:llama";
+			case mtMagmaCube:      return "minecraft:magma_cube";
+			case mtMooshroom:      return "minecraft:mooshroom";
+			case mtMule:           return "minecraft:mule";
+			case mtOcelot:         return "minecraft:ocelot";
+			case mtPanda:          return "minecraft:panda";
+			case mtParrot:         return "minecraft:parrot";
+			case mtPhantom:        return "minecraft:phantom";
+			case mtPig:            return "minecraft:pig";
+			case mtPiglin:         return "minecraft:piglin";
+			case mtPiglinBrute:    return "minecraft:piglin_brute";
+			case mtPillager:       return "minecraft:pillager";
+			case mtPolarBear:      return "minecraft:polar_bear";
+			case mtPufferfish:     return "minecraft:pufferfish";
+			case mtRabbit:         return "minecraft:rabbit";
+			case mtRavager:        return "minecraft:ravager";
+			case mtSalmon:         return "minecraft:salmon";
+			case mtSheep:          return "minecraft:sheep";
+			case mtShulker:        return "minecraft:shulker";
+			case mtSilverfish:     return "minecraft:silverfish";
+			case mtSkeleton:       return "minecraft:skeleton";
+			case mtSkeletonHorse:  return "minecraft:skeleton_horse";
+			case mtSlime:          return "minecraft:slime";
+			case mtSnowGolem:      return "minecraft:snow_golem";
+			case mtSpider:         return "minecraft:spider";
+			case mtSquid:          return "minecraft:squid";
+			case mtStray:          return "minecraft:stray";
+			case mtStrider:        return "minecraft:strider";
+			case mtTraderLlama:    return "minecraft:trader_llama";
+			case mtTropicalFish:   return "minecraft:tropical_fish";
+			case mtTurtle:         return "minecraft:turtle";
+			case mtVex:            return "minecraft:vex";
+			case mtVillager:       return "minecraft:villager";
+			case mtVindicator:     return "minecraft:vindicator";
+			case mtWanderingTrader:return "minecraft:wandering_trader";
+			case mtWitch:          return "minecraft:witch";
+			case mtWither:         return "minecraft:wither";
+			case mtWitherSkeleton: return "minecraft:wither_skeleton";
+			case mtWolf:           return "minecraft:wolf";
+			case mtZoglin:         return "minecraft:zoglin";
+			case mtZombie:         return "minecraft:zombie";
+			case mtZombieHorse:    return "minecraft:zombie_horse";
+			case mtZombiePigman:   return "minecraft:zombie_pigman";
```

Comment:
Mojang changed the name of the zombie pigman in 1.16 to zombified piglin. Which name should we choose for saving?

Reply:
The ZPM will never be forgotten! But change happens, we should follow vanilla and move to ZPLs.